packages/web-js-sdk/src/enhancers/withPersistTokens/helpers.ts

```diff
@@ -34,40 +34,46 @@ function setJwtTokenCookie(
 
 export const persistTokens = (
   { refreshJwt, sessionJwt, ...cookieParams } = {} as Partial<JWTResponse>,
-  sessionTokenViaCookie = false
+  sessionTokenViaCookie = false,
+  storagePrefix = ''
 ) => {
   // persist refresh token
-  refreshJwt && setLocalStorage(REFRESH_TOKEN_KEY, refreshJwt);
+  refreshJwt &&
+    setLocalStorage(`${storagePrefix}${REFRESH_TOKEN_KEY}`, refreshJwt);
 
   // persist session token
   if (sessionJwt) {
     sessionTokenViaCookie
       ? setJwtTokenCookie(SESSION_TOKEN_KEY, sessionJwt, cookieParams)
-      : setLocalStorage(SESSION_TOKEN_KEY, sessionJwt);
+      : setLocalStorage(`${storagePrefix}${SESSION_TOKEN_KEY}`, sessionJwt);
   }
 };
 
 /** Return the refresh token from the localStorage. Not for production usage because refresh token will not be saved in localStorage. */
-export function getRefreshToken() {
-  return getLocalStorage(REFRESH_TOKEN_KEY) || '';
+export function getRefreshToken(prefix: string = '') {
+  return getLocalStorage(`${prefix}${REFRESH_TOKEN_KEY}`) || '';
```

Comment:
if we decide not to expose prefix as an argument -  we can simply use the prefix value local storage here, rather then pass it down the chain